models/sushiswap/avalanche_c/sushiswap_avalanche_c_trades.sql

```diff
@@ -11,25 +11,25 @@
 {% set project_start_date = '2021-03-15' %}
 
 WITH sushiswap_dex AS (
-    SELECT  t.evt_block_time                                             AS block_time,
-            `to`                                                         AS taker,
+    SELECT t.evt_block_time         AS block_time,
+            to                                                           AS taker,
             sender                                                       AS maker,
-            CASE WHEN amount0Out = 0 THEN amount1Out ELSE amount0Out END AS token_bought_amount_raw,
-            CASE WHEN amount0In = 0 THEN amount1In ELSE amount0In END    AS token_sold_amount_raw,
-            cast(NULL as double)                                         AS amount_usd,
-            CASE WHEN amount0Out = 0 THEN token1 ELSE token0 END         AS token_bought_address,
-            CASE WHEN amount0In = 0 THEN token1 ELSE token0 END          AS token_sold_address,
+            case when amount0Out = UINT256 '0' then amount1Out else amount0Out end as token_bought_amount_raw,
```

Comment:
nit: For ease of reviewing it's better to keep the existing casing when possible (no need to fix it here or elsewhere, but good to keep in mind for future unmigrated models)